src/bonefish/messages/wamp_call_message.hpp

```diff
@@ -185,7 +185,7 @@ inline void wamp_call_message::set_options(const msgpack::object& options)
 
 inline void wamp_call_message::set_procedure(const std::string& procedure)
 {
-    m_procedure = procedure;
+    m_procedure = msgpack::object(procedure, get_zone());
```

Comment:
`set_procedure()` doesn't seem to be called from anywhere, should we delete these functions instead? As it stands, I can't even judge why it should be initialized with or without rezoning, because there are no users that would use it in any way.